io/plugins/eu.esdihumboldt.hale.io.topojson.test/src/eu/esdihumboldt/hale/io/topojson/test/TopoJsonInstanceWriterTest.groovy

```diff
@@ -144,8 +155,9 @@ class TopoJsonInstanceWriterTest {
 			assertEquals(1, json.objects.Topology.geometries[1].arcs.size())
 			assertEquals(3, json.objects.Topology.geometries[1].arcs[0].size())
 			assertEquals([-1, 2, 3], json.objects.Topology.geometries[1].arcs[0])
-		}
 
+			assertNull(json.objects.Topology.geometries[1].'properties'.label)
+			assertNull(json.objects.Topology.geometries[1].'properties'.date)
```

Comment:
Please also check the case where a value should be present, to test that that value is not replaced by null unintentionally.